pkgs/development/libraries/simpleitk/default.nix

```diff
@@ -4,12 +4,10 @@ stdenv.mkDerivation rec {
   pname = "simpleitk";
   version = "2.2.1";
 
-  outputs = [ "out" "dev" ];
-
```

Comment:
This means more likely the cake file is wrong

Reply:
CMake builds rely on package configuration (`.cmake`) files in their dependencies, and with a `dev` output the resulting SimpleITK package configuration file tells depending packages to expect shared libraries in the `/nix/store/xxxxx-simpleitk-dev`, which is wrong.